test/evergreen.yml

```diff
@@ -1088,3 +1234,60 @@ buildvariants:
     - name: unit-test
     - name: fops
 
+- name: little-endian
+  display_name: Little-endian (x86)
+  run_on:
+  - ubuntu1404-test
+  batchtime: 1440 # 1 day
+  expansions:
+    smp_command: -j $(grep -c ^processor /proc/cpuinfo)
+    configure_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH
+    test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs
+  tasks:
+  - name: compile
+  - name: generate-datafile-little-endian
+    distros:
+    - ubuntu1404-build
+  - name: dump-datafile-little-endian
+    distros:
+    - ubuntu1404-build
+  - name: dump-datafile-from-big-endian
+    distros:
+    - ubuntu1404-build
+
+- name: big-endian
+  display_name: Big-endian (s390x/zSeries)
+  modules:
+  - enterprise
+  run_on:
+  - ubuntu1604-zseries-small
+  batchtime: 1440 # 1 day
+  stepback: false
+  expansions:
+    compile_flags: --ssl MONGO_DISTMOD=ubuntu1604 --release -j3 CCFLAGS="-march=z196 -mtune=zEC12" --variables-files=etc/scons/mongodbtoolchain_gcc.vars
```

Comment:
I don't understand why this would be necessary? Similarly for goroot and a few others below - do we need them?

Reply:
These settings were copied over from `mongodb-mongo-master` s390x build variant, and should be unnecessary to include here. Will remove them to see if a new patch build breaks.